Resources/Textures/Clothing/Head/Misc/cthulhu.rsi/meta.json

```diff
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from https://github.com/brainfood1183/space-station-14",
```

Comment:
You need to show exact commit like this:
```
Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/547852588166c8e091b441e4e67169e156bb09c1
```

But I would suggest you to add more simpler copyright that used for other original works. For example:
```
"Veritius#2351 (257233913951289344) on discord / @Veritius on GitHub"
```

Reply:
thanks, i wasnt really sure what to put for copyright.